src/libs/common.go

```diff
@@ -477,7 +489,16 @@ func WriteCiliumPolicyToYamlFile(namespace string, policies []types.CiliumNetwor
 }
 
 func WriteKubeArmorPolicyToYamlFile(fname string, policies []types.KubeArmorPolicy) {
-	fileName := GetEnv("POLICY_DIR", "./")
+
+	policyDir := cfg.CurrentCfg.ConfigSysPolicy.SystemPolicyDir
+	var fileName string
+
+	if policyDir != "./" {
+		fileName = GetEnv("POLICY_DIR", "./")
+	} else {
+		fileName = policyDir
+	}
+
```

Comment:
This code does not make sense.
If the SystemPolicyDir in the configmap is set to any value other thant "./" then it will not be used!!

Reply:
No Rahul. As per implementation, if it is any value other than **./**, then the same will be used.